src/pages/stacks-blockchain/running-testnet-node.md

```diff
@@ -52,26 +52,35 @@ git clone https://github.com/blockstack/stacks-blockchain.git; cd stacks-blockch
 Install the Stacks node by running:
 
 ```bash
-cargo build --workspace --bin stacks-node
-# binary will be in target/debug/stacks-node
-./target/debug/stacks-node krypton
-# release build
 cargo build --workspace --release --bin stacks-node
 # binary will be in target/release/stacks-node
 ```
+To install Stacks node with extra debugging symbols, run:
+
+```bash
+cargo build --workspace --bin stacks-node
+# binary will be in target/debug/stacks-node krypton
+```
 
 -> This process will take a few minutes to complete
 
 ## Step 2: Running the node
 
 You're all set to run a node that connects to the testnet network.
 
-Back in the command line, run:
+If installed without debugging symbols, run:
 
 ```bash
-stacks-node krypton
+target/release/stacks-node krypton
 ```
 
+If installed with debugging symbols, run:
+
+```bash
+target/debug/stacks-node krypton
+```
+
+
```

Comment:
can you run the linter on this? I think the double empty line will fail the deployment